src/components/index.ts

```diff
@@ -48,6 +48,7 @@ export {
   TooltipProps,
   Validation,
   Icon,
+  InputDecorator,
```

Comment:
can't we group all that's input related in a single export?

Reply:
It's already the case, InputDecorator, TexField and TextArea are exported in symphony-bdk-ui-toolkit-components/src/components/input/index.ts
Then in this file, we re-export all the components at the root level.